### PR TITLE
Fix typo in Chainlink external adapter tutorial

### PR DESCRIPTION
**Pull request by PoCk3T**
I believe a typo was introduced when writing [ "How to Configure and Use Your Own Chainlink Node And External Adapter In Your Avalanche dApp"](https://docs.avax.network/community/tutorials-contest/2021/red-dev-avalanche-chainlink-tutorial) tutorial.

The writer clearly stated '_Create a .api file to expose credentials for the API and GUI interfaces_' but then proceeds to create credentials in the .env file instead.

I'm assuming it's an honest mistake coming from a copy/paste of the previous command described in the tutorial

### CODE REVIEW

File: docs/community/tutorials-contest/2021/red-dev-avalanche-chainlink-tutorial/README.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ DATABASE_URL=postgresql://$USERNAME:$PASSWORD@$HOST:5432/chainlink?sslmode=disab
 Create a .api file to expose credentials for the API and GUI interfaces:
 ```
 $ echo "youremailaddress@yourcompany.com
-$PASSWORD" > ~/.chainlink-avalanche/.env
+$PASSWORD" > ~/.chainlink-avalanche/.api
```
Comment (cgcardona) [inline docs/community/tutorials-contest/2021/red-dev-avalanche-chainlink-tutorial/README.md:207]: Actually it should be the opposite. All instances of `.api` should be changed to `.env`. `.api` is a photoshop format.

**Comment by PoCk3T** [comment]
I understand the best practice of using .env for all environment variables, but then some part of the tutorial will need to be completely re-written @cgcardona 

Except this typo, it's otherwise a very good tutorial, something other blockchains don't document as well as Ava Labs did; 

it's worth going ahead with this typo fix for now and think about a larger re-writing of the tutorial later on IMHO

**Comment by cgcardona** [comment]
What parts do you think need to be rewritten? Can you make the changes required to change `.api` -> `.env` and i'll merge this PR. Then we can talk about making the other changes.

**Comment by PoCk3T** [comment]
The following command in the tutorial is looking explicitly for a dedicated .password file and a dedicated .api file:

`$ docker run -d --name chainlink-avalanche-node -p 6688:6688 -v ~/.chainlink-avalanche:/chainlink -it --env-file=/root/.chainlink-avalanche/.env smartcontract/chainlink:0.10.3 local n -p /chainlink/.password -a /chainlink/.api`

Deleting those files and moving their content to the .env file is going to break it.

**Comment by cgcardona** [comment]
Ok let's hold off on merging this for now. Let me confirm the correct syntax and I'll ping you in this PR when I confirm. Thx

**Review by cgcardona — APPROVED** [review]

**Pull request merged**